python/setup.py.in

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+"""
+Setup for SWIG Python bindings for lttoolbox
+"""
+from os import path
+from setuptools import Extension, setup
+from distutils.command.build import build
+
+
+class CustomBuild(build):
+    sub_commands = [
+        ('build_ext', build.has_ext_modules),
+        ('build_py', build.has_pure_modules),
+        ('build_clib', build.has_c_libraries),
+        ('build_scripts', build.has_scripts),
+    ]
+
+
+def get_sources():
+    sources = ['analysis_wrap.cpp']
+    cc_sources = ['alphabet.cc', 'compression.cc', 'fst_processor.cc', 'lt_locale.cc',
+                  'node.cc', 'state.cc', 'trans_exe.cc', 'xml_parse_util.cc']
+    rel_path = '@top_srcdir@/lttoolbox/'
+    sources.extend(path.join(rel_path, f) for f in cc_sources)
+    return sources
+
+analysis_module = Extension(
+    name='_analysis',
+    sources=get_sources(),
+    include_dirs=['@top_srcdir@', '/usr/include/libxml2'],
+    library_dirs=['/usr/include/libxml2'],
+    extra_compile_args='@CXXFLAGS@'.split(),
+    extra_link_args=['-lxml2'],
+)
+
+setup(
+    name='python-@PACKAGE@',
```

Comment:
Is there a reason we need to prepend `python-` here? It's clearly a python package already.